portfolio/pom.xml

```diff
@@ -23,6 +23,11 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
Doesn't necessarily apply here. But it's normally a good idea to keep things sorted, e.g. in alphabetical order.

Reply:
I'll keep that in mind from now on, thank you!

Reply:
I sorted the dependencies & I'll keep that tip in mind from now on, thank you!